_docs/tasks/telemetry/fluentd.md

```diff
@@ -0,0 +1,405 @@
+---
+title: Logging with Fluentd
+
+overview: This task shows you how to configure Istio to log to a Fluentd daemon
+
+order: 60
+
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+This task shows how to configure Istio to create custom log entries
+and send them to a Fluentd daemon. At the end of this task, a new log
+stream will be enabled sending logs to an example Fluentd /
+Elasticsearch / Kibana stack.
+
+The [BookInfo]({{home}}/docs/guides/bookinfo.html) sample application is used
+as the example application throughout this task.
+
+## Before you begin
+* [Install Istio]({{home}}/docs/setup/) in your cluster and deploy an
+  application. This task assumes that Mixer is setup in a default configuration
+  (`--configDefaultNamespace=istio-system`). If you use a different
+  value, update the configuration and commands in this task to match the value.
+
+## Setup Fluentd
+
+In your cluster, you may already have a Fluentd DaemonSet running,
+such the add-on described
+[here](https://kubernetes.io/docs/tasks/debug-application-cluster/logging-elasticsearch-kibana/)
+and
+[here](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/fluentd-elasticsearch),
+or something specific to your cluster provider. This is likely
+configured to send logs to an Elasticsearch system or logging
+provider.
+
+You may use these Fluentd daemons, or any other Fluentd daemon you
+have set up, as long as they are listening for forwarded logs, and
+Istio's Mixer is able to connect to them. In order for Mixer to connect to a
+running Fluentd daemon, you may need to add a
+[service](https://kubernetes.io/docs/concepts/services-networking/service/). The
+Fluentd configuration to listen for forwarded logs is:
+
+```
+<source>
+  type forward
+</source>
+```
+
+The full details of connecting Mixer to all possible Fluentd
+configurations is beyond the scope of this task.
+
+### Example Fluentd, Elasticsearch, Kibana Stack
+
+For the purposes of this task, you may deploy the example stack
+provided. Save the following as `logging-stack.yaml`.
```

Comment:
can we do two things to help the reader:
1) Before the `Save the following...` can we add some text like:
> The example stack below consists of the following: a new namespace (logging), an ElasticSearch service and deployment, a Kibana service and deployment, and a Fluentd service, daemon deployment, and configuration for forwarding logs to the ElasticSearch backend.
2) Add comments to each subsection of the configuration.

Another option would be to add a section about "Understanding what was done" or something.